lib/src/visitor/perform.dart

```diff
@@ -509,11 +519,17 @@ class _PerformVisitor
     });
   }
 
+  Uri _resolvePackageUri(Uri packageUri) {
+    return packageUri.scheme == 'package' && packageResolver != null
+        ? packageResolver.resolveUri(packageUri) ?? new Uri()
```

Comment:
I don't think `new Uri()` is right here. A blank URL doesn't help anyone. We should throw a useful error message saying "Can't resolve "$packageUri"" if it fails to resolve, or if there's no resolver available.